suite-native/navigation/src/components/Screen.tsx

```diff
@@ -9,17 +9,19 @@ import { BottomTabBarHeightContext } from '@react-navigation/bottom-tabs';
 
 import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
 import { Color, nativeSpacings } from '@trezor/theme';
-import { Box, Divider } from '@suite-native/atoms';
 import { selectIsAnyBannerMessageActive } from '@suite-common/message-system';
+import { Box } from '@suite-native/atoms';
 
-import { ScreenContent } from './ScreenContent';
+import { ScreenContentWrapper } from './ScreenContentWrapper';
+import { ScreenHeader } from './ScreenHeader';
 
 type ScreenProps = {
     children: ReactNode;
-    header?: ReactNode;
     footer?: ReactNode;
-    hasDivider?: boolean;
+    subheader?: ReactNode;
```

Comment:
nit: maybe subheader -> subHeader to stay consistent?

Reply:
If you're not dogmatic about it, I would keep it. Subheader is a word so no need to camelCase it.